src/features/profile/mySecurity/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Please use components in common folder

Reply:
this is not global component, it only uses for my security page.